includes/admin/admin-pages.php

```diff
@@ -279,6 +279,11 @@ function edd_is_admin_page( $passed_page = '', $passed_view = '', $include_non_e
 						$found = true;
 					}
 					break;
+				case 'marketing':
```

Comment:
Yes I also agree it's horrendous. Something that's also bothered me for months is that `'reports'` is in there twice.

I am in favour of refactoring, but maybe >= 3.0 because I think 3.0 touched this too and the merge conflicts would kill me. :sweat_smile: